From ac8149d104de5ac3fcb17c3de1dab654e04a7952 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Tue, 12 Sep 2017 12:12:19 +0100
Subject: [PATCH] Resolves: tdf#106163 safe fix for wrong final row/col
 selection in rtl
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change-Id: I773de03768b0a5b28e6b4f63bd65dd270dc975cd
Reviewed-on: https://gerrit.libreoffice.org/42197
Reviewed-by: Caolán McNamara <caolanm@redhat.com>
Tested-by: Caolán McNamara <caolanm@redhat.com>
(cherry picked from commit 65b756f6058697ccb787506c3cc32e1e818bcc37)
---
 svx/source/tbxctrls/layctrl.cxx | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/svx/source/tbxctrls/layctrl.cxx b/svx/source/tbxctrls/layctrl.cxx
index b0e707f..d0bcb9c 100644
--- a/svx/source/tbxctrls/layctrl.cxx
+++ b/svx/source/tbxctrls/layctrl.cxx
@@ -161,6 +161,8 @@ void TableWindow::dispose()
 void TableWindow::MouseMove( const MouseEvent& rMEvt )
 {
     SfxPopupWindow::MouseMove( rMEvt );
+    if (IsInCleanUp())
+        return;
     Point aPos = rMEvt.GetPosPixel();
     Point aMousePos( aPos );
 
-- 
2.9.5

